### ISSUE TITLE
Default state for equipment

### ISSUE THREAD

**Issue by kevinmic** [reef-pi/reef-pi#719]
I have noticed that if I power cycle my pi while a macro is running it leaves the equipment in the state it was last in before the pi reset. It would be nice instead to have a "default" state such that when the reef-pi boots it initializes all equipment into its "default" state.

**Comment by ranjib** [comment]
This is not a normal power cycle right? You are not triggering proper service shutdown, this is a hard power reset. Its hard to ensure reef-pi's behavior if its hard reset while a macro is running. Macro's are currently triggered manually, on-demand. So I'd expect users to expect abnormal behavior in such cases (hard power reset + macro running). If the equipment is under some other control mechanism (like ato or temperature controller) then it should be auto-corrected.

**Comment by kevinmic** [comment]
Yes, this is not a normal power cycle.  I unplugged it while the macro was running to see what would happen. I was testing the macro to see if I could do dosing through it instead. As part of my testing I wanted to make sure that if I hard power cycled the pi that the pump wouldn't be running when it rebooted. When it turned back on it was still in the state it was when it turned off, thus I abandoned the idea.

I had assumed that if it was happening on the macro then it was happening elsewhere. When I setup my auto top off I will have to check again.
